solidity-v1/dashboard/src/components/modal/threshold/WithdrawGrantedTokens.jsx

```diff
@@ -0,0 +1,205 @@
+          The following tokens are available to be released and withdrawn from
+          your token grant.
+        </p>
+        <div>
+          <OnlyIf condition={grants.length > 1}>
+            <div className="flex row center space-between">
+              <h4 className="text-grey-70">Withdraw:</h4>
+              <span className="text-grey-70 small">
+                Total:{" "}
+                <TokenAmount
+                  amount={totalReadyToRelease}
+                  token={KEEP}
+                  wrapperClassName="withdraw-granted-tokens__total-ready-to-release-amount"
+                  amountClassName="text-grey-60"
+                  symbolClassName="text-grey-60"
+                />
+              </span>
+            </div>
+          </OnlyIf>
+          <form>
+            <Accordion
+              allowZeroExpanded={grants.length > 1}
+              className={
+                "withdraw-granted-tokens__grants-accordion withdraw-granted-tokens__grants-accordion-container"
+              }
+              preExpanded={grants.length === 0 ? [`grant-${grants[0].id}`] : []}
+            >
+              {grants
+                .slice(0, numberOfGrantsDisplayed)
+                .map((grant) =>
+                  renderGrant(
+                    grant,
+                    grants.length,
+                    selectedGrant,
+                    setSelectedGrant
+                  )
+                )}
+            </Accordion>
+          </form>
+          <OnlyIf condition={grants.length > numberOfGrantsDisplayed}>
+            <Button
+              className="withdraw-granted-tokens__view-more-button"
+              onClick={() =>
+                setNumberOfGrantsDisplayed(
+                  numberOfGrantsDisplayed + DEFAULT_GRANTS_TO_DISPLAY
+                )
+              }
+            >
+              <Icons.Add />
+              &nbsp;View More
+            </Button>
+          </OnlyIf>
+        </div>
+      </ModalBody>
+      <ModalFooter>
+        <SubmitButton
+          className="btn btn-primary btn-lg mr-2"
+          type="submit"
+          onSubmitAction={onWithdrawClick}
+          disabled={grants.length > 1 && !selectedGrant}
+        >
+          withdraw
+        </SubmitButton>
+        <Button className={`btn btn-unstyled text-link`} onClick={onClose}>
+          Cancel
+        </Button>
+      </ModalFooter>
+    </>
+  )
+})
+
+const renderGrant = (
+  grant,
+  totalNumberOfGrants,
+  selectedGrant,
+  setSelectedGrant
+) => {
+  return (
+    <AccordionItem uuid={`grant-${grant.id}`} key={`grant-${grant.id}`}>
+      <input
+        className="radio-without-label"
+        type="radio"
+        name="selectedGrantId"
+        value={grant.id}
+        id={`grant-${grant.id}`}
+        checked={selectedGrant?.id === grant?.id}
+        onChange={() => {
+          setSelectedGrant(grant)
+        }}
+      />
+      <label htmlFor={`grant-${grant.id}`} />
```

Comment:
We still display the radio button if there is only one grant.

![image](https://user-images.githubusercontent.com/57687279/142647675-bed8782d-a112-492a-afd4-2cff08bd1209.png)

Reply:
Fixed in 62d9677b091a4c97b7cb6dddc999564a6d50b046